### PR TITLE
Fix link

### PR DESCRIPTION
**Pull request by jmealing**
FR: https://github.com/canada-ca/systeme-conception/pull/330

### CODE REVIEW

**Comment by jmealing** [comment]
@delisma does not resolve it, same issue

File: style-guide/index.md [hunk 1]
```diff
@@ -3207,7 +3207,7 @@ title: "Canada.ca Content Style Guide"
     <li><cite><a href="https://www.noslangues-ourlanguages.gc.ca/en/writing-tips-plus/index-eng">Writing Tips Plus</a></cite>, Resource of the Language Portal of Canada, Government of Canada</li>
   </ul>
 </section>
-<aside class="wb-fnote wb-init wb-fnote-inited" role="note" id="wb-auto-4">
+<aside class="wb-fnote" role="note">
 	<h3 id="fn">Footnotes</h3>
 	<dl>
 		<dt id="fn1-dt">Footnote 1</dt>
```
Comment (delisma) [inline style-guide/index.md]:
```suggestion
		<dt>Footnote 1</dt>
```

Comment (delisma) [reply style-guide/index.md]: L3216
```
<a href="#fn1-rf">
```

**Review by delisma — CHANGES_REQUESTED** [review]
L2410, L3213 and L3216 to modify

File: style-guide/index.md (hunk 1 above)
Comment (delisma) [reply style-guide/index.md]: L2410 has a stray open em tag
```
<p>Flesch-Kincaidreading level**<em>**: </cite>5.9</p>
```

Comment (jmealing) [reply style-guide/index.md]: @delisma I'm not sure where the `em/cite` it's supposed to be as this wasn't in the scope of work I did. I would assume to remove both stray tags as it isn't in a previous mention of the same text above

File: style-guide/index.md [hunk 2]
```diff
@@ -3207,12 +3207,12 @@ title: "Canada.ca Content Style Guide"
...
 </section>
-<aside class="wb-fnote wb-init wb-fnote-inited" role="note" id="wb-auto-4">
+<aside class="wb-fnote" role="note">
 	<h3 id="fn">Footnotes</h3>
 	<dl>
-		<dt id="fn1-dt">Footnote 1</dt>
+		<dt>Footnote 1</dt>
 		<dd id="fn1" tabindex="-1">
-			<p>International Plain Language Federation: <a href="​​https://www.iplfederation.org/plain-language">​https://www.iplfederation.org/plain-language</a></p>
+			<p>International Plain Language Federation:<br><a href="​​https://www.iplfederation.org/plain-language">​https://www.iplfederation.org/plain-language</a></p>
```
Comment (delisma) [inline style-guide/index.md]:
```suggestion
			<p>International Plain Language Federation:<br><a href="https://www.iplfederation.org/plain-language">​https://www.iplfederation.org/plain-language</a></p>
```
This should fix it. I've removed the 2 white-spaces before the https

File: style-guide/index.md [hunk 3]
```diff
@@ -3207,12 +3207,12 @@ title: "Canada.ca Content Style Guide"
...
-<aside class="wb-fnote wb-init wb-fnote-inited" role="note" id="wb-auto-4">
+<aside class="wb-fnote" role="note">
 	<h3 id="fn">Footnotes</h3>
 	<dl>
-		<dt id="fn1-dt">Footnote 1</dt>
+		<dt>Footnote 1</dt>
 		<dd id="fn1" tabindex="-1">
-			<p>International Plain Language Federation: <a href="​​https://www.iplfederation.org/plain-language">​https://www.iplfederation.org/plain-language</a></p>
+			<p>International Plain Language Federation:<br><a href="https://www.iplfederation.org/plain-language">​https://www.iplfederation.org/plain-language</a></p>
 			<p class="fn-rtn small"><a href="#fn1-0-rf" data-wb-fnote="true"><span class="wb-inv">Return to footnote </span>1<span class="wb-inv"> referrer</span></a></p>
```
Comment (delisma) [inline style-guide/index.md]: Last change I promise.
```suggestion
			<p class="fn-rtn small"><a href="#fn1-rf"><span class="wb-inv">Return to footnote </span>1<span class="wb-inv"> referrer</span></a></p>
```

**Review by delisma — CHANGES_REQUESTED** [review]
🚀 pre-approved. Waiting for the french repo to be updated too.

**Comment by jmealing** [comment]
@delisma FR has been created https://github.com/canada-ca/systeme-conception/pull/330

File: style-guide/index.md [hunk 4]
```diff
@@ -3207,13 +3207,13 @@ title: "Canada.ca Content Style Guide"
     <li><cite><a href="https://www.noslangues-ourlanguages.gc.ca/en/writing-tips-plus/index-eng">Writing Tips Plus</a></cite>, Resource of the Language Portal of Canada, Government of Canada</li>
   </ul>
 </section>
-<aside class="wb-fnote wb-init wb-fnote-inited" role="note" id="wb-auto-4">
+<aside class="wb-fnote" role="note">
 	<h3 id="fn">Footnotes</h3>
 	<dl>
-		<dt id="fn1-dt">Footnote 1</dt>
+		<dt>Footnote 1</dt>
 		<dd id="fn1" tabindex="-1">
```
Comment (delisma) [inline style-guide/index.md]:
```suggestion
		<dd id="fn1">
```

**Review by delisma — CHANGES_REQUESTED** [review]

**Comment by jmealing** [comment]
@delisma I accepted your changes so not sure why it's still flagging a change request

**Review by delisma — APPROVED** [review]
🚀 LGTM

**Pull request merged**